nodes/parachain/src/service.rs

```diff
@@ -20,12 +20,19 @@
 use std::{sync::Arc, time::Duration};
 
 use cumulus_client_cli::CollatorOptions;
+#[cfg(not(feature = "async-backing"))]
 // Local Runtime Types
 use polimec_runtime::{
 	opaque::{Block, Hash},
 	RuntimeApi,
 };
 
+#[cfg(feature = "async-backing")]
+use politest_runtime::{
```

Comment:
If we enable the `async-backing` feature we have to use the `politest_runtime` since it implements a `RuntimeApi` needed by the `task_manager`.